internal/pithosctl/pkg/pithos/healthz.go

```diff
@@ -142,6 +142,12 @@ func (h *Healthz) createObject() error {
 }
 
 func (h *Healthz) GetObject() error {
-	_, err := h.Client.GetObject(h.Bucket, objectName, minio.GetObjectOptions{})
+	reader, err := h.Client.GetObject(h.Bucket, objectName, minio.GetObjectOptions{})
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	var data []byte
+	_, err = reader.Read(data)
```

Comment:
Do you need to read the object's contents or can you do `reader.Stat` instead - to query the metadata? Does it matter if you read the contents?

Reply:
I believe in the case of Stat it is using only something like HEAD(I know it is S3, not HTML, just analogy). I want to be sure the object is readable, not just its metadata.